lib/reporters/tap.js

```diff
@@ -125,7 +125,7 @@ class TapReporter {
 		this.reportStream.write(`# ${stripAnsi(title)}${os.EOL}`);
 		if (evt.logs) {
 			for (const log of evt.logs) {
-				const logLines = indentString(log, 4).replace(/^ {4}/, '  # ');
+				const logLines = indentString(log, 4).replace(/^ {4}/gm, '#   ');
```

Comment:
Do we need extra indentation?

Reply:
Well it was there before, and makes output more readable. Also the same indentation is in `supertap`